repository/src/main/java/com/epam/esm/repository/repositoryinterfaces/Repository.java

```diff
@@ -16,15 +18,15 @@
      * Fina a tag or certificate by its unique Id
      *
      * @param id id of a tag or certificate
-     * @return {@link Optional} of {@link Tag} or {@link Certificate} entity.
+     * @return {@link Optional} of {@link Tag}, {@link Certificate}, {@link Order} or {@link User} entity.
```

Comment:
You don't know ALL implementation of your interface. You don't need to list of possible return values here

Reply:
fixed